contribute/testing.md

```diff
@@ -0,0 +1,230 @@
+# Beam Testing Documentation
```

Comment:
Adding the following to the top of the file will fix the tests:

---
layout: default
title: 'Beam Testing Guide'
redirect_from: '/docs/testing'
---